pkg/asset/installconfig/baremetal/validation.go

```diff
@@ -0,0 +1,23 @@
+package baremetal
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/baremetal/validation"
+)
+
+// Validate performs platform validation specifically for any optional requirement
+// to be called when the cluster creation takes place
+func Validate(ic *types.InstallConfig) error {
```

Comment:
this function name doesn't convey the special nature of validation of install-config only when creating cluster.

Reply:
Thanks. I agree. Will change.